src/PatrolDetailView/index.js

```diff
@@ -94,8 +94,6 @@ const PatrolDetailView = () => {
   const [notesToAdd, setNotesToAdd] = useState([]);
   const [showInvalidDatesModal, setShowInvalidDatesModal] = useState(false);
 
-  const patrolTracker = trackEventFactory(PATROL_DETAIL_VIEW_CATEGORY);
```

Comment:
We had this tracker repeated in this file. I just left the one that is defined outside of the component so we don't redefine it on every render.